app/models/vacancy.rb

```diff
@@ -67,13 +73,22 @@ class Vacancy < ApplicationRecord
   validate :enable_job_applications_cannot_be_changed_once_listed
 
   has_noticed_notifications
+  has_paper_trail on: [:update], only: ATTRIBUTES_TO_TRACK
 
   before_save :on_expired_vacancy_feedback_submitted_update_stats_updated_at
   before_save :refresh_geolocation, if: -> { job_location_changed? }
 
   EQUAL_OPPORTUNITIES_PUBLICATION_THRESHOLD = 5
   EXPIRY_TIME_OPTIONS = %w[7:00 8:00 9:00 10:00 11:00 12:00 13:00 14:00 15:00 16:00 17:00 23:59].freeze
 
+  def self.array_enums
+    {
+      job_roles: { "teacher" => 0, "leadership" => 1, "teaching_assistant" => 6, "education_support" => 4, "sendco" => 5, "send_responsible" => 2, "ect_suitable" => 3 },
```

Comment:
It's good that this is explicit now, but I think you can simplify it a bit and not type out the explicit values:

```ruby
def self.array_enums
  {
    job_roles: job_roles,
    key_stages: key_stages,
    working_patterns: working_patterns,
  }
end
```